src/dash-table/components/CellFactory.tsx

```diff
@@ -138,22 +159,52 @@ export default class CellFactory {
         (o, c) => o.length ? o.concat(c) : c
     ));
 
+    getDataOpCell = memoizerCache<[number, number]>()((
+        operation: JSX.Element,
+        style: CSSProperties | undefined,
+        borderBottom: any,
+        borderLeft: any,
+        borderRight: any,
+        borderTop: any
```

Comment:
The current `edges` resolution always results in new objects, making caching impossible for many scenarios. Break it down into the various borders allows for caching.

This is a caching layer on the cells themselves, as `getDataOpCells` often can't provide meaningful caching at the "table" level.

This is rather ham fisted but works within the current structure.